### PR TITLE
Summarize build results by printing a JSON object

### PR DESCRIPTION
**Pull request by angelonakos**
*Issue #, if available:*

*Description of changes:*

A flag was added to `run-build` that summarizes the results of a build by printing a JSON object.
    
A user will be able to get a breakdown of statuses across pipelines as well as a breakdown of the
status of each pipeline.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: lib/post_build.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+
+def get_summary(run_dict):
+    # The list of possible statuses can be viewed in the man page for run.json
+    status_summary_map = {}
+    pipeline_status_map = {}
+    for pipeline in run_dict["pipelines"]:
+        status_name = pipeline["status"]
+        status_name_pretty = status_name.title().replace("_", " ")
+        if status_name_pretty not in status_summary_map:
```
Comment (karkhaz) [inline lib/post_build.py]:
```
try:
  status_summary_map[status_name_pretty] += 1
except KeyError:
    status_summary_map[status_name_pretty] = 1

**Review by karkhaz — CHANGES_REQUESTED** [review]
I don't think this application-level reporting belongs here. Different users of Litani would want different kinds of reports, and I wouldn't want to bake this particular JSON report into the CL interface... once it's in, we can't change it, even if somebody wants something slightly different.

I think applications should just run `dump-run` or `get-jobs` (preferably the latter) and use that for whatever output they want.

File: lib/post_build.py (hunk 1 above)
Comment (angelonakos) [reply lib/post_build.py]: Done

**Comment by angelonakos** [comment]
Canceling

**Pull request closed without merging**